src/components/userComponents/emailPopup/page.tsx

```diff
@@ -0,0 +1,198 @@
+                  
+                      >
+                      <VscClose 
+                          className="text-4xl text-[#808080] pt-[17px] pr-[17px]"
+                          onClick={closeModal}/>
+                      </button>
+                    </div>
+
+                    {subscribed ? (
+                      <><div className="absolute pt-[52px] top-[72px] left-1/2 transform -translate-x-1/2 -translate-y-1/2">
+                    <PiSealCheck className="text-[#3F6A38] text-5xl" />
+                      </div>
+                  
+                      <h2 className="pt-[108px] pl-[32px] pr-[32px] text-center font-Lato font-bold text-[19px] text-[#3B3B3B]">
+                        THANKS FOR SUBSCRIBING!
+                      </h2>
+                      
+                      <p className="text-center text-sm font-Lato text-[#3B3B3B] pt-[10px] pl-[30px] pr-[30px] text-[16px]">
+                      Your sign-up request was successful! Please check your email inbox to confirm.</p>
+                      
+
+                    <Link href="/collectionsPage" >
+
+                      <div className="flex items-center justify-center pt-[24px]">
+                      <button
+                        type="button"
+                        className="bg-[#7CA24E] w-[283px] h-[43px] text-white rounded-2xl p-[21px] mt-[8px] flex items-center justify-center"
+                      >
+                        Back to Home
+                      </button>
+                    </div>
+                    </Link>
+
+                      </>
+                    ) : (
+                      <>
+
+                  <div className="absolute top-[102px] left-1/2 transform -translate-x-1/2 -translate-y-1/2">
+                  <PiPaperPlaneTiltBold className="text-[#3F6A38] text-4xl" />
+                  </div>
+
+                  <h2 className="pt-[108px] pl-[48.5px] pr-[48.5px] text-center font-Lato font-bold text-[19px] text-[#3B3B3B]">
+                    JOIN OUR NEWSLETTER!
+                  </h2>
+                  <p className="text-center text-sm font-Lato text-[#3B3B3B] pt-[10px] pl-[30px] pr-[30px] text-[16px]">
+                    Monthly updates on our work and involvement opportunities.
+                  </p>
+
+
+                  <div className="flex items-center justify-center pt-[15px]">
+                    <input
+                      type="text"
+                      placeholder="Enter your email"
+                      value={inputValue}
+                      onChange={handleChange}
+                      className={`font-Lato text-sm bg-gray-100 border rounded-md pl-[13px] w-[283px] h-[43px] items-center ${
+                        inputValue ? 'text-black' : 'text-[#BDBDBD]'
+                      }`}
+                    />
+                  </div>
+
+                  <div className="flex items-center justify-center">
+                    <button
+                      type="button"
```

Comment:
readability: lets also move this to be a helper component that we call in this segment.  Will be much easier to maintain this way.  I would also recommend switching the rendering cases here -> This should be the first thing that renders based on if !subscribed instead.